lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -85,6 +85,76 @@ class StandardAccountTest {
         StandardAccount(Felt.ZERO, privateKey, provider, chainId)
     }
 
+    @Test
+    fun `creating cairo 0 account with automatic version determination`() {
+        val call = Call(
+            contractAddress = balanceContractAddress,
+            entrypoint = "increase_balance",
+            calldata = listOf(Felt(10), Felt(20), Felt(30)),
+        )
+        val accountDetailsCairo0 = devnetClient.createDeployAccount(classHash = DevnetClient.accountContractClassHashCairo0, accountName = "cairo0_account").details
+        val accountAddressCairo0 = accountDetailsCairo0.address
+        val signerCairo0 = StarkCurveSigner(accountDetailsCairo0.privateKey)
```

Comment:
It doesn't matter for this test what signer will be used. We can simply reuse the default `signer` or pass any value as a private key.

Reply:
Good remark 👍